runtime/src/pallets_governance.rs

```diff
@@ -0,0 +1,184 @@
+
+use crate::{
+    constants, Call, Event, FinancialCommittee, Origin, OriginCaller, RootCommittee, Runtime,
+    Scheduler, TechnicalCommittee,
+};
+use frame_support::parameter_types;
+use nodle_chain_primitives::{AccountId, BlockNumber};
+use sp_core::u32_trait::{_1, _2};
+use sp_runtime::ModuleId;
+pub use sp_runtime::{Perbill, Perquintill};
+
+// Shared parameters with all collectives / committees
+parameter_types! {
+    pub const MotionDuration: BlockNumber = 2 * constants::DAYS;
+    pub const MaxProposals: u32 = 100;
+    pub const MaxMembers: u32 = 50;
+}
+
+pub type TechnicalCollective = pallet_collective::Instance2;
+impl pallet_collective::Config<TechnicalCollective> for Runtime {
+    type Origin = Origin;
+    type Proposal = Call;
+    type Event = Event;
+    type MotionDuration = MotionDuration;
+    type MaxProposals = MaxProposals;
+    type WeightInfo = ();
+    type MaxMembers = MaxMembers;
+    type DefaultVote = pallet_collective::PrimeDefaultVote;
+}
+
+// --- Financial committee
+pub type FinancialCollective = pallet_collective::Instance3;
+impl pallet_collective::Config<FinancialCollective> for Runtime {
+    type Origin = Origin;
+    type Proposal = Call;
+    type Event = Event;
+    type MotionDuration = MotionDuration;
+    type MaxProposals = MaxProposals;
+    type WeightInfo = ();
+    type MaxMembers = MaxMembers;
+    type DefaultVote = pallet_collective::PrimeDefaultVote;
+}
+
+// --- Technical committee
+
+impl pallet_membership::Config<pallet_membership::Instance1> for Runtime {
+    type Event = Event;
+    type AddOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, RootCollective>;
+    type RemoveOrigin =
+        pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, RootCollective>;
+    type SwapOrigin =
+        pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, RootCollective>;
+    type ResetOrigin =
+        pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, RootCollective>;
+    type PrimeOrigin =
+        pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, RootCollective>;
+    type MembershipInitialized = TechnicalCommittee;
+    type MembershipChanged = TechnicalCommittee;
+}
+
+impl pallet_membership::Config<pallet_membership::Instance3> for Runtime {
+    type Event = Event;
+    type AddOrigin = pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, RootCollective>;
+    type RemoveOrigin =
+        pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, RootCollective>;
+    type SwapOrigin =
+        pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, RootCollective>;
+    type ResetOrigin =
+        pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, RootCollective>;
+    type PrimeOrigin =
+        pallet_collective::EnsureProportionMoreThan<_1, _2, AccountId, RootCollective>;
+    type MembershipInitialized = FinancialCommittee;
+    type MembershipChanged = FinancialCommittee;
+}
```

Comment:
Let's move this up to the `Financial committee` section

Reply:
Done :+1: